ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx

```diff
@@ -120,66 +121,78 @@ export function TimeSeriesChartOptionsEditor(props: TimeSeriesChartOptionsEditor
   };
```

Comment:
I think it would be a good idea to do the change to this file in a separate, quick PR as a follow-up. Otherwise, I could see some nasty merge conflict hell because several other people on the team are touching this chunk of react code. Leaving it in this draft, for now, so people can see what the shape of the props looks like and give feedback.